lib/testing_api_onlyoffice_com/test_instance/main_page/document_builder_api/document_builder_getting_started.rb

```diff
@@ -0,0 +1,48 @@
+require 'httparty'
+require 'onlyoffice_file_helper'
+module TestingApiOnlyfficeCom
+  # https://user-images.githubusercontent.com/40513035/55968227-48e64600-5c84-11e9-8ca6-90d1057918cd.png
+  # /docbuilder/gettingstarted
+  class DocumentBuilderGettingStarted
+    include PageObject
+
+    # actions
+    link(:linuxx64, xpath: '//a[@href="http://download.onlyoffice.com/install/desktop/docbuilder/documentbuilder-x64.tar.gz"]')
+    link(:windowsx64, xpath: '//a[@href="http://download.onlyoffice.com/install/desktop/docbuilder/documentbuilder-x64.zip"]')
```

Comment:
Better not to include full path (which can changed) in xpath
Better one is
`//a[contains(@href, 'documentbuilder-x64.zip')]`